### ISSUE TITLE
Reactivity in condensed phase: putting higher weights on solutes' atoms compared to solvents' atoms in the training loss.

### ISSUE THREAD

**Issue by ines-art** [ACEsuit/mace#1339]
Hi, 

Working on reactivity in condensed phase (explicit solvation), I noticed that when scratch training or fine-tuning a mace model, I get higher RMSE(Force) and RMSE(Energy) for the reactants and products (solutes) compared to the solvent molecules, which makes sense since there are much more solvent molecules in the training dataset compared to the solutes.

Therefore, would it be possible to add a functionality in mace training (scratch training and fine-tuning) to be able to put **higher weights in the training loss for certain groups of atoms** (reactants and products in this case, that could be identified e.g. by the order in which they appear in the extxyz) compared to other atoms (solvent ones)?

Thanks a lot for your help!

**Comment by RivasAntonio** [comment]
Hey @ines-art I think that you could try the argument config_type_weights:'{"reactants":XX, "products":YY}'. Make sure that in the info of each configuration you assign the corresponding config_type tag.

**Comment by ines-art** [comment]
Hi @RivasAntonio! Thanks for the answer. The thing is that in one configuration, the reactant or product is solvated in a bunch of solvent molecules. So even if we put higher weights in those configurations compared to some where we only have solvents, the loss is dominated by the solvent anyway. But maybe it is worth trying that, hoping that the model learns better about the solutes (reactant and product). And do you know if it is possible to put some weights on certain atoms?

**Comment by ilyes319** [comment]
yep in principle this quite easy to add because all the code is there to do that, if someone wants to have a go. It is just a matter of reading the right entry.
